src/index.css

```
@@ -442,6 +442,44 @@
     border-radius: 2px;
   }
 
+  /* Tag styling — shares the WikiLink color scheme so existing/referenced/ghost
+     states are visually consistent across both mark types. See issue #725. */
+  /* タグのスタイル。WikiLink と同じ色系列を共有し、existing / referenced /
+     ghost の 3 状態を両マーク共通で表現する。Issue #725 を参照。*/
```

Stylelint (17.9.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)